ova/configs/config.json

```diff
@@ -6,7 +6,7 @@
             "scheme": "http",
             "organization": "dispatch",
             "insecure": true,
-            "api-https-port": 443
+            "apihttpsport": 443
```

Comment:
is this key correct?  I can't find reference to it in the source

Reply:
Yes, it must no dash
@zimengyang find this